includes/revisions-control/class-significant-revision.php

```diff
@@ -0,0 +1,204 @@
+<?php
+/**
+ * A class to handle significant revisions checks and actions
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+use WP_Post;
+
+/**
+ * Significant Revision Object
+ */
+class Significant_Revision {
+
+	/**
+	 * The name of the post meta that stores the significant revisions IDs
+	 *
+	 * @var string
+	 */
+	const RELEVANT_IDS_META_KEY = '_significant_revision';
```

Comment:
"Relevant" and "significant" are being used here for the same thing, but IMHO a more appropriate term for the whole thing is "kept". "Kept revisions" is a more direct term for what it does.

Reply:
I'd say `protected` or `locked` are even better. "Significant" is what has been proposed and used in the front end so that's what I used... should we ask around to see what's the best name for it?